gulpfile.js

```diff
@@ -36,7 +36,7 @@ gulp.task('mocha', function(done) {
 
 // new run tests
 gulp.task('karma', function() {
-  return gulp.src(paths.test)
+  return gulp.src('./foobar') // 
```

Comment:
apparently there is a bug with gulp karma where the karma.config file is not read.  a work around is to set the parameter to something arbitrary. see link below:

http://stackoverflow.com/questions/22413767/angular-testing-with-karma-module-is-not-defined